Source/WebCore/bindings/js/JSDOMConvertStrings.cpp

```diff
@@ -154,6 +154,7 @@ String trustedTypeCompliantString(TrustedType expectedType, JSGlobalObject& glob
     auto stringValue = expectedType == TrustedType::TrustedScriptURL
         ? Converter<IDLUSVString>::convert(global, input)
         : Converter<IDLDOMString>::convert(global, input);
+    RETURN_IF_EXCEPTION(throwScope, { });
```

Comment:
This line was needed else it triggered the a [test failure](https://ews-build.s3-us-west-2.amazonaws.com/macOS-AppleSilicon-Sonoma-Debug-WK2-Tests-EWS/abbfbb81-14986/js/dom/missing-exception-check-in-convertVariadicArguments-crash-log.txt)